package.json

```diff
@@ -10,46 +10,46 @@
     "web": "TAMAGUI_TARGET=web expo start --web"
   },
   "dependencies": {
-    "@emotion/native": "^11.11.0",
-    "@emotion/react": "^11.11.1",
-    "@gluestack-style/animation-plugin": "latest",
-    "@gluestack-style/react": "latest",
-    "@gluestack-ui/provider": "latest",
-    "@gluestack-ui/vstack": "latest",
-    "@shopify/restyle": "^2.4.2",
-    "@tamagui/animations-react-native": "^1.47.1",
-    "@tamagui/config": "^1.47.1",
-    "@tamagui/core": "^1.47.1",
-    "@tamagui/font-inter": "^1.47.1",
-    "@tamagui/react-native-media-driver": "^1.47.1",
-    "@tamagui/shorthands": "^1.47.1",
-    "@tamagui/theme-base": "^1.47.1",
-    "@tamagui/themes": "^1.47.1",
-    "@types/react": "~18.2.14",
-    "babel-plugin-transform-inline-environment-variables": "^0.4.4",
-    "dripsy": "^4.3.3",
-    "expo": "~49.0.5",
-    "expo-font": "^11.4.0",
-    "expo-splash-screen": "~0.20.4",
-    "expo-status-bar": "~1.6.0",
-    "gluestack-ui": "^0.4.8",
-    "nativewind": "^2.0.11",
+    "@emotion/native": "11.11.0",
+    "@emotion/react": "11.11.0",
```

Comment:
@atanaskanchev thanks for the PR, any reason why this patch was downgraded: "@emotion/react": "11.11.0",? It's just for version alignment with emotion/native?

Reply:
Hi @efstathiosntonas my bad, pushed @emotion/react": "11.11.1